support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -13,13 +13,12 @@ case class SendThankYouEmailState(
   user: User,
   giftRecipient: Option[GiftRecipient],
   product: ProductType,
-  paymentMethod: Either[PaymentMethod, RedemptionData],
+  paymentMethod: Either[(PaymentMethod, PaymentSchedule), RedemptionData],
```

Comment:
Can you explain this change please?

Reply:
yes, the payment schedule only exists where there are payments, if it's a free sub there's no concept of a payment schedule

Reply:
I think I commented elsewhere that a tuple isn't so clear and maybe a case class would be more descriptive, if you agree?

Reply:
I'm not too sure about this new model, it seems as though it would be clearer to retain the existing `paymentMethod` type and make `paymentSchedule` an Option. 
I realise that that is slightly less tight from a data validity point of view though. Maybe we could introduce a wrapper type `PaymentMethodWithSchedule` in place of this tuple?

Reply:
We overlapped with our above comments, case class seems like a good solution to me